### PR TITLE
feat: Create profile screen

### LINKED ISSUE #18: feat/ui-create-profile-screen
**AC**
Prepare profile screen according to the mockups (https://www.figma.com/file/RkzObeqWU4mYZwuYVnnPOC/TeamUp?node-id=52412%3A15900)

Just UI, you can fill it with some dummy data for now

### PR DESCRIPTION
**Pull request by KonTra7cf**

### CODE REVIEW

**Review by Mariusz-web — APPROVED** [review]

File: src/components/Profile/desktop.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { Avatar, Box, Typography } from '@mui/material';
+
+import { Description } from './description';
+
+export const DesktopProfile = () => {
```
Comment (Veryyapeee) [inline src/components/Profile/desktop.tsx]: Do you have to split that logic for two separate components?

File: src/components/Profile/UserProfile.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { useOurMedia } from '../../hooks';
+import { DesktopProfile } from './desktop';
+import { MobileProfile } from './mobile';
+
+export const UserProfile = () => {
```
Comment (Veryyapeee) [inline src/components/Profile/UserProfile.tsx]: You should add TODO comment and create issue for conntecting that screen to API

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/components/Profile/UserProfile.tsx (hunk 2 above)
Comment (Veryyapeee) [reply src/components/Profile/UserProfile.tsx]: I don't see created issue, which should be linked in here

Also please answer in the conversation always after making changes

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

**Review by maciejopalinski — CHANGES_REQUESTED** [review]
I fixed the integration with API, but profile looks bad.
User avatar should not be streched. Just crop it or keep the aspect ratio.

image

**Comment by KonTra7cf** [comment]
@PoProstuMieciek - I've added property to crop the image so now it should be fine

**Review by maciejopalinski — APPROVED** [review]
I removed the desktop layout as it was not working. Also I simplified components structure. Approved!

**Review by T0ffik — APPROVED** [review]

**Review by Veryyapeee — APPROVED** [review]

**Pull request merged**